cg-spec/editors_draft.html

```diff
@@ -404,11 +407,23 @@ <h2>Triples and occurrences</h2>
         -->
       </pre>
 
-    </section>
+      <p>With this solution RDF-star takes three statements to annotate an occurrence. In contrast to RDF-star's approach RDF standard reification is defined as referring to occurrences. It is however underspecified insofar as it doesn't provide a way to denote the location - some graph, document or other source of a set of triples - in which a statement occurs, as the ":in" property in the example above does. RDF standard reification needs three statements to describe a triple and would need a forth statement to specify its location. Only then a fifth statement would be able to annotate such an occurrence. Note that the ":occurrenceOf" and ":in" properties in the example above serve only illustrative purposes as standardizing a proper vocabulary to denote occurrences is still an <a href="#rdf-star-vocabulary">open issue</a>. By comparison RDF/XML's <a href="https://www.w3.org/TR/rdf-syntax-grammar/#section-Syntax-reifying">ID</a> attribute implements reification without any deviation from RDF's abstract syntax and semantics and enables statement annotation without the need for any extra statements.</p>
```

Comment:
> RDF standard reification is defined as referring to occurrences

Technically, this is not true. RDF standard reification is not "defined as referring" to anything. The semantics does not impose to them to denote any particular thing (since the reification predicate have no particular semantics themselves).

What is true is that standard reification bnodes lend themselves better than embedded triples to be interpreted as occurrences, because many of them can exist with the same subject, predicate and object (unlike embedded triples). But actually the semantics says nothing about what precisely they refer to.

But stating that, *by definition*, standard reification denotes occurrences, is wrong.

Reply:
"[ ...] the reification describes the relationship between a token of a triple and the resources that the triple refers to." [RDF 1.1 Semantics ](https://www.w3.org/TR/rdf11-mt/#reification)
Whatever is true "technically" I think my portrayel of the topic captures the essence, but I do of course welcome any proposal of an also technically correct re-wording.

Reply:
I stand corrected on this.

I keep thinking of reification as largely underspecified (and since this section you quoted is non-normative, that's still partially true), but at least the intent is clearly stated there. My bad.